onebusaway-android/src/main/java/org/onebusaway/android/tripservice/PollerTask.java

```diff
@@ -156,14 +158,16 @@ private Long checkArrivals(ObaArrivalInfoResponse response, Cursor c) {
                 // when calculated with the *predicted* arrival time
                 // is past now.
                 long depart = info.getPredictedArrivalTime();
+                boolean isArriving = true;
                 if (depart == 0) {
                     depart = info.getScheduledArrivalTime();
+                    isArriving = false;
```

Comment:
To keep our arrival/departure labeling consistent with the normal arrival times lists, let's construct an instance of `ArrivalInfo` here from the `ObaArrivalInfo`, and then call `arrivalInfo.isArrival()` to determine if it's an arrival or departure.

This is actually a little more complicated than I was originally thinking, though, as previously we were always using arrival time for the reminder - technically I think we should probably be reminding users with the departure times for departures, and arrival time for arrivals.  I think we could use `arrivalInfo.getEta()` for this (internally we already calculate ETA from arrival or departure appropriately), but the resolution for `getEta()` is currently in seconds, and this method expects to return milliseconds.  Maybe we add another method that returns ETA in milliseconds.